### DISCUSSION TITLE
New Unenrollment Method [NOT SKID FRIENDLY, VERY ADVANCED]

### DISCUSSION THREAD

**Discussion by Su4o** [3kh0/ext-remover#821] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
![image](https://github.com/3kh0/ext-remover/assets/58097612/ce7406e1-e41d-41a5-bf99-b6369ac73167)

[linked YouTube video](https://www.youtube.com/watch?v=9L9qWoDa420)

It’s called cryptosmite. DO NOT UPDATE TO CHROMEOS 120, USE CAUB! (or flag)

**Note: This is a very advanced method and not for people who just started exploiting. Please be cautious...**

**Comment by 3kh0** [comment]
bro is on his ipad too lol

**Reply by Lytonicity** [reply]
1. What is the link to this discord server?
2. How Do I do the exploit?

**Reply by Su4o** [reply]
>:)
![C0F360DC-2AE8-4DBA-AC56-07F5F1027FA8](https://github.com/3kh0/ext-remover/assets/127699325/0e4fa858-6485-427c-9075-647354e3b563)

**Reply by Su4o** [reply]
@Lytonicity its discord.gg/unblock

and also, this exploit isn’t here yet, it may come soon

**Reply by Lytonicity** [reply]
thx dude

**Reply by crllect** [reply]
sick, wonder how it's gonna work, if its anything like sh1mmer, definitely gonna stay tuned for the write-up.

**Reply by kxtzownsu** [reply]
as a developer for cryptosmite (can confirm), i did NOT want this on 3kh0's repo, also its not skid friendly so it doesnt need to be on here (this repo is for like... people *just* starting with chromebook bypassing)

**Reply by Su4o** [reply]
yeah, but well what do you want me to do? Should i close, I can't delete the poll. @kxtzownsu

**Reply by Su4o** [reply]
also, it's not really a tut, just a heads up for a new exploit.

**Reply by funkymokey** [reply]
itd be funny if people complained though

**Reply by MunyDev** [reply]
Wow, stop putting my unenrollment method in 3kho's repo
💀

**Reply by Su4o** [reply]
Ok, but what am I supposed to do? Should I edit the message and title. @MunyDev

**Reply by MunyDev** [reply]
Edit the message so it says very advanced, because it is very advanced.

**Reply by Su4o** [reply]
Bet. Sure

**Reply by MunyDev** [reply]
Cryptosmite came out

**Comment by kxtzownsu** [comment]
https://github.com/FWSmasher/Cryptosmite

**Comment by xoidex** [comment]
i decided to powerwash my chromebook for a sprite
totally worth it
